spec/brcobranca/retorno_cnab_240_spec.rb

```diff
@@ -18,6 +18,7 @@
     expect(pagamento.valor_titulo).to eql('000000000034400')
     expect(pagamento.banco_recebedor).to eql('001')
     expect(pagamento.agencia_recebedora_com_dv).to eql('020850')
+    expect(pagamento.data_ocorrencia).to eql('02012012')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.